axelor-account/src/main/java/com/axelor/apps/account/service/fixedasset/AbstractFixedAssetLineComputationServiceImpl.java

```diff
@@ -485,4 +488,12 @@ protected BigDecimal getAccountingValue(FixedAssetLine fixedAssetLine) {
         ? fixedAssetLine.getCorrectedAccountingValue()
         : fixedAssetLine.getAccountingValue();
   }
+
+  protected LocalDate getFirstServiceDate(FixedAsset fixedAsset) {
```

Comment:
This method is never used, maybe use it everywhere you do a :
```
    LocalDate firstServiceDate =
        fixedAsset.getFirstServiceDate() == null
            ? fixedAsset.getAcquisitionDate()
            : fixedAsset.getFirstServiceDate();
```
And set it as a public static method, so you don't have to inject the service all the time? 
Or just delete this method